tests/dialects/test_bigquery.py

```diff
@@ -252,6 +252,12 @@ def test_bigquery(self):
                 "bigquery": "CREATE TABLE db.example_table (x INT64) PARTITION BY x CLUSTER BY x",
             },
         )
+        self.validate_all(
+            "CREATE TABLE db.example_table (x BIGNUMERIC) PARTITION BY x cluster by x",
+            write={
+                "bigquery": "CREATE TABLE db.example_table (x BIGNUMERIC) PARTITION BY x CLUSTER BY x",
+            },
+        )
```

Comment:
Let's use `validate_identity` here instead (note: `cluster by` needs to be converted to uppercase).

Reply:
ACK. I will change this.